recipe/meta.yaml

```
@@ -1,60 +1,77 @@
 {% set name = "hmmlearn" %}
-{% set version = "0.2.8" %}
+{% set version = "0.3.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 696931e3dce6801cc9b78c629f543061dd7eb67a71155b03d1b39ea172a2a5f9
+  sha256: edaf485fdb1ea88da9ac642b2006c63d9950dd15d4d132f7205305d383e6f745
 
 build:
-  number: 1
-  skip: true  # [py<35]
+  number: 0
+  skip: true  # [py<38]
   script:
-    - {{ PYTHON }} setup.py build_ext
-    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv  # [unix]
-    - {{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt  # [win]
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -vv
+  missing_dso_whitelist:  # [s390x]
+    - '$RPATH/ld64.so.1'  # [s390x]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler("cxx") }}
   host:
     - pip
     - python
-    - numpy
     - pybind11 >=2.6
     - setuptools
-    - setuptools_scm
+    - setuptools_scm >=3.3
+    - pytest
```

```suggestion
    - pytest
```
I'm not seeing why we add `pytest` here. I thought it was only needed for the tests. Am I missing something?

---

My mistake, I was trying to figure out how to get the tests to run without building the package twice.